clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/security/DockerRegistryNamedAccountCredentials.groovy

```diff
@@ -31,9 +31,27 @@ public class DockerRegistryNamedAccountCredentials implements AccountCredentials
   public DockerRegistryNamedAccountCredentials(String accountName, String environment, String accountType,
                                                String address, String username, String password, String email,
                                                List<String> repositories, List<String> requiredGroupMembership) {
+    if (!accountName || accountName.length() == 0) {
```

Comment:
Can just be `if (!accountName)`.

http://groovy-lang.org/semantics.html#_strings